lib/esc.js

```diff
@@ -249,11 +235,11 @@ util.inherits(ESC, Emitter);
  * @param  {Float} speed 0...100 (full range)
  *
  * @return {ESC} instance
+ * @deprecated Will be deleted in version 1.0.0. Use throttle(us) instead.
  */
-
-ESC.prototype.speed = function(speed) {
+/* istanbul ignore next */
+ESC.prototype.speed = util.deprecate(function(speed) {
```

Comment:
This whole function is trash. It did async work to speed up/down, which was really just papering over the fact that integers between 0-100 scales to ~18us for each step and that's awful, so let's get rid of this trash.